packages/core-data/src/hooks/use-entity-records.ts

```diff
@@ -97,8 +108,35 @@ export default function useEntityRecords< RecordType >(
 		[ kind, name, queryAsString, options.enabled ]
 	);
 
```

Comment:
I noticed that this hook is using a special `useQuerySelect`. That felt a bit too magic to me, I'd have preferred as simple useSelect I think but it's not that important.